microservice/src/test/java/de/gsi/microservice/utils/SystemProperties.java

```diff
@@ -0,0 +1,57 @@
+package de.gsi.microservice.utils;
+
+import static java.util.Map.Entry;
+
+import java.util.Properties;
+import java.util.Set;
+
+public class SystemProperties {
```

Comment:
Codacy found an issue: [The utility class name 'SystemProperties' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper|Constants)'](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=6307742)